upup/pkg/fi/cloudup/awstasks/loadbalancer_attributes.go

```diff
@@ -166,6 +166,9 @@ func (_ *LoadBalancer) modifyLoadBalancerAttributes(t *awsup.AWSAPITarget, a, e,
 	if e.ConnectionSettings != nil && e.ConnectionSettings.IdleTimeout != nil {
 		request.LoadBalancerAttributes.ConnectionSettings.IdleTimeout = e.ConnectionSettings.IdleTimeout
 	}
+	if e.CrossZoneLoadBalancing != nil && e.CrossZoneLoadBalancing.Enabled != nil {
```

Comment:
Should this be an else off the if statement on line 136?

Reply:
Ah yep 👍